TESTING_GUIDE.md

```diff
@@ -53,7 +53,7 @@ Set up an OpenSearch cluster with Docker:
 ```bash
 export OPENSEARCH_PASSWORD=<<your_password>>
 cd tests/default
-docker compose up -d
+docker-compose up -d
```

Comment:
This is incorrect, old version, https://docs.docker.com/compose/releases/migrate/.